examples/refrigerator-app/silabs/README.md

```diff
@@ -0,0 +1,374 @@
+    established, look for the RTT log
+
+    ```
+        [DL] Device Role: CHILD
+        [DL] Partition Id:0x6A7491B7
+        [DL] \_OnPlatformEvent default: event->Type = 32778
+        [DL] OpenThread State Changed (Flags: 0x00000001)
+        [DL] Thread Unicast Addresses:
+        [DL]    2001:DB8::E1A2:87F1:7D5D:FECA/64 valid preferred
+        [DL]    FDDE:AD00:BEEF::FF:FE00:2402/64 valid preferred rloc
+        [DL]    FDDE:AD00:BEEF:0:383F:5E81:A05A:B168/64 valid preferred
+        [DL]    FE80::D8F2:592E:C109:CF00/64 valid preferred
+        [DL] LwIP Thread interface addresses updated
+        [DL] FE80::D8F2:592E:C109:CF00 IPv6 link-local address, preferred)
+        [DL] FDDE:AD00:BEEF:0:383F:5E81:A05A:B168 Thread mesh-local address, preferred)
+        [DL] 2001:DB8::E1A2:87F1:7D5D:FECA IPv6 global unicast address, preferred)
+    ```
+
+    (you can verify that the device is on the thread network with the command
+    `router table` using a serial terminal (screen / minicom etc.) on the board
+    running the refrigerator-app example. You can also get the address list with
+    the command ipaddr again in the serial terminal )
+
+    You can provision the Chip device using Chip tool Android or iOS app or
+    through CLI commands on your OT BR
+
+    The
+    [CHIPTool](https://github.com/project-chip/connectedhomeip/blob/master/examples/chip-tool/README.md)
+    can now be used to send ZCL commands to the refrigerator device. For
+    instance, to set the refrigerator covering lift by percentage:
+
+    ```
+    chip-tool pairing ble-thread 1 hex:<operationalDataset> 20202021 3840
+
+    ./chip-tool refrigeratoralarm read state 1 1
+
+    Refrigerator Endpoint Id 1
+
+    This enpoint support the refrigerator alarm cluster
+
+    ./chip-tool temperaturecontrol read temperature-setpoint 1 2
+
+    Enpoint for temperaturecontrol is 2 and 3
+
+    Cold Cabinet Endpoint Id 2
+    Freeze Cabinet Endpoint Id 3
+
+    This enpoint support Temperature control cluster
+    ```
+
+    To see the supported refrigerator cluster commands, use:
+
+    ```
+    chip-tool refrigeratorandtemperaturecontrolledcabinetmode
+    chip-tool temperaturecontrol
+    chip-tool refrigeratoralarm
+
+    ```
+
+### Notes
+
+-   Depending on your network settings your router might not provide native ipv6
+    addresses to your devices (Border router / PC). If this is the case, you
+    need to add a static ipv6 addresses on both device and then an ipv6 route to
+    the border router on your PC
+
+          # On Border Router :
+          $ sudo ip addr add dev <Network interface> 2002::2/64
+
+          # On PC (Linux) :
+          $ sudo ip addr add dev <Network interface> 2002::1/64
+
+          # Add Ipv6 route on PC (Linux)
+          $ sudo ip route add <Thread global ipv6 prefix>/64 via 2002::2
+
+## OTA Software Update
+
+For the description of Software Update process with EFR32 example applications
```

Comment:
should we have something common for SiWx917 ?

Reply:
we don't have the document in pointing to the process of the OTA update for SiWx917. i will update when we have the document

Reply:
Added OTA documentation SiWx917 board